src/infrastructure/database/models/reported_error/reported_error_database_handler.py

```diff
@@ -45,11 +45,10 @@ async def delete_reported_error(collection: Collection[ReportedError], error_id:
     @staticmethod
     async def create_reported_error(
             error_collection: Collection[ReportedError],
-            user_collection: Collection[User],
             user_id: ObjectId,
-            payload: ReportedErrorCreate
+            username: str,
+            payload: ReportedErrorCreate,
     ) -> None:
-        username = user_collection.find_one({'_id': user_id})
```

Comment:
Czemu rezygnujemy z tego? 
Zgłoszeń raczej nie będzie dużo a tak to będzie transparentne dla Damiana.

Reply:
Nie musimy szukać username w bazie bo przekazujemy go od razu:
 await DatabaseHandler.create_reported_error(
        db.reported_errors, current_user['_id'], **current_user['username']**, reported_error_create_payload
    )

Reply:
kk